plugins/poll.py

```diff
@@ -89,4 +98,4 @@ def vote(self, msg):
         else:
             return msg.reply("Tu as déjà voté", hilight=True)
 
-        msg.reply(self.print_poll())
+        msg.reply("{}: tu as voté pour {}".format(msg.user, self.votes[index]['name']))
```

Comment:
You can use `msg.reply("Tu as voté pour " + self.votes[index]['name'], hilight=True)` instead of manually prefixing with the nick